<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<title>CRAN - Package plinkQC</title>
<link rel="stylesheet" type="text/css" href="../../CRAN_web.css" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="citation_title" content="Genotype Quality Control with 'PLINK' [R package plinkQC version 0.2.0]" />
<meta name="citation_author" content="Hannah Meyer" />
<meta name="citation_publication_date.Published" content="2018-11-09" />
<meta name="citation_public_url" content="https://CRAN.R-project.org/package=plinkQC" />
<meta name="DC.identifier" content="https://CRAN.R-project.org/package=plinkQC" />
<meta name="DC.publisher" content="Comprehensive R Archive Network (CRAN)" />
<style type="text/css">
  table td { vertical-align: top; }
</style>
</head>
<body>
<h2>plinkQC: Genotype Quality Control with 'PLINK'</h2>
<p>Genotyping arrays enable the direct measurement of an individuals
    genotype at thousands of markers. 'plinkQC' facilitates genotype quality
    control for genetic association studies as described by Anderson and
    colleagues (2010) &lt;<a href="https://doi.org/10.1038/nprot.2010.116">doi:10.1038/nprot.2010.116</a>&gt;. It makes 'PLINK' basic
    statistics (e.g. missing genotyping rates per individual, allele frequencies
    per genetic marker) and relationship functions accessible from 'R' and
    generates a per-individual and per-marker quality control report.
    Individuals and markers that fail the quality control can subsequently be
    removed to generate a new, clean dataset. Removal of individuals based on
    relationship status is optimised to retain as many individuals as possible
    in the study.</p>
<table summary="Package plinkQC summary">
<tr>
<td>Version:</td>
<td>0.2.0</td>
</tr>
<tr>
<td>Depends:</td>
<td>R (&ge; 3.3.0)</td>
</tr>
<tr>
<td>Imports:</td>
<td>methods, <a href="../optparse/index.html">optparse</a>, <a href="../data.table/index.html">data.table</a> (&ge; 1.11.0), <a href="../R.utils/index.html">R.utils</a>, <a href="../ggplot2/index.html">ggplot2</a>, <a href="../ggforce/index.html">ggforce</a>, <a href="../ggrepel/index.html">ggrepel</a>, <a href="../cowplot/index.html">cowplot</a>, <a href="../UpSetR/index.html">UpSetR</a>, <a href="../dplyr/index.html">dplyr</a>, <a href="../sys/index.html">sys</a></td>
</tr>
<tr>
<td>Suggests:</td>
<td><a href="../testthat/index.html">testthat</a>, <a href="../knitr/index.html">knitr</a>, <a href="../rmarkdown/index.html">rmarkdown</a></td>
</tr>
<tr>
<td>Published:</td>
<td>2018-11-09</td>
</tr>
<tr>
<td>Author:</td>
<td>Hannah Meyer <a href="https://orcid.org/0000-0003-4564-0899"><img alt="ORCID iD" src="/web/orcid.svg" style="width:16px; height:16px; margin-left:4px; margin-right:4px; vertical-align:middle" /></a> [aut, cre]</td>
</tr>
<tr>
<td>Maintainer:</td>
<td>Hannah Meyer  &#x3c;&#x68;&#x61;&#x6e;&#x6e;&#x61;&#x68;&#x20;&#x61;&#x74;&#x20;&#x65;&#x62;&#x69;&#x2e;&#x61;&#x63;&#x2e;&#x75;&#x6b;&#x3e;</td>
</tr>
<tr>
<td>BugReports:</td>
<td><a href="https://github.com/HannahVMeyer/plinkQC/issues">https://github.com/HannahVMeyer/plinkQC/issues</a></td>
</tr>
<tr>
<td>License:</td>
<td><a href="../../licenses/MIT">MIT</a> + file <a href="LICENSE">LICENSE</a></td>
</tr>
<tr>
<td>URL:</td>
<td><a href="https://github.com/HannahVMeyer/plinkQC">https://github.com/HannahVMeyer/plinkQC</a></td>
</tr>
<tr>
<td>NeedsCompilation:</td>
<td>no</td>
</tr>
<tr>
<td>SystemRequirements:</td>
<td>plink (1.9)</td>
</tr>
<tr>
<td>Materials:</td>
<td><a href="readme/README.html">README</a> <a href="news/news.html">NEWS</a> </td>
</tr>
<tr>
<td>CRAN&nbsp;checks:</td>
<td><a href="../../checks/check_results_plinkQC.html">plinkQC results</a></td>
</tr>
</table>
<h4>Downloads:</h4>
<table summary="Package plinkQC downloads">
<tr>
<td> Reference&nbsp;manual: </td>
<td> <a href="plinkQC.pdf"> plinkQC.pdf </a> </td>
</tr>
<tr>
<td>Vignettes:</td>
<td>
<a href="vignettes/AncestryCheck.pdf">AncestryCheck</a><br/>
<a href="vignettes/Genomes1000.pdf">1000Genomes</a><br/>
<a href="vignettes/HapMap.pdf">HapMap</a><br/>
<a href="vignettes/plinkQC.pdf">GenotypeQC_with_plinkQC</a><br/>
</td>
</tr>
<tr>
<td> Package&nbsp;source: </td>
<td> <a href="../../../src/contrib/plinkQC_0.2.0.tar.gz"> plinkQC_0.2.0.tar.gz </a> </td>
</tr>
<tr>
<td> Windows&nbsp;binaries: </td>
<td> r-devel: <a href="../../../bin/windows/contrib/3.6/plinkQC_0.2.0.zip">plinkQC_0.2.0.zip</a>, r-release: <a href="../../../bin/windows/contrib/3.5/plinkQC_0.2.0.zip">plinkQC_0.2.0.zip</a>, r-oldrel: <a href="../../../bin/windows/contrib/3.4/plinkQC_0.2.0.zip">plinkQC_0.2.0.zip</a> </td>
</tr>
<tr>
<td> OS&nbsp;X&nbsp;binaries: </td>
<td> r-release: <a href="../../../bin/macosx/el-capitan/contrib/3.5/plinkQC_0.2.0.tgz">plinkQC_0.2.0.tgz</a>, r-oldrel: <a href="../../../bin/macosx/el-capitan/contrib/3.4/plinkQC_0.2.0.tgz">plinkQC_0.2.0.tgz</a> </td>
</tr>
<tr>
<td> Old&nbsp;sources: </td>
<td> <a href="https://CRAN.R-project.org/src/contrib/Archive/plinkQC"> plinkQC archive </a> </td>
</tr>
</table>
<h4>Linking:</h4>
<p>Please use the canonical form
<a href="https://CRAN.R-project.org/package=plinkQC"><samp>https://CRAN.R-project.org/package=plinkQC</samp></a>
to link to this page.</p>
</body>
</html>
